docs/contributing.md

```diff
@@ -35,6 +35,7 @@ Here's a typical lifecycle of events to contribute to Hudi.
  - [Optional] If you want to get involved, but don't have a project in mind, please check JIRA for small, quick-starters.
  - [Optional] Familiarize yourself with internals of Hudi using content on this page, as well as [wiki](https://cwiki.apache.org/confluence/display/HUDI)
  - Once you finalize on a project/task, please open a new JIRA or assign an existing one to yourself. (If you don't have perms to do this, please email the dev mailing list with your JIRA id and a small intro for yourself. We'd be happy to add you as a contributor)
+ - Almost all PRs should be linked to a JIRA. Before you begin work, click "Start Progress" on the JIRA, which tells everyone that you are working on the issue actively.
```

Comment:
Maybe we could add a good practice like this

> Before you start working, if you are unsure about the solution, you can seek a committer as a mentor and participate in the discussion. 

I think this is a more efficient way of working with the community.

Reply:
https://hudi.apache.org/contributing.html already covers this IMO?

Reply:
yes